static/data-ingestion.yaml

```
@@ -916,90 +586,25 @@ paths:
           required: false
           schema:
             type: string
-            enum: [gzip]
+            enum: [ gzip ]
           description: Use this header if the payload is compressed with gzip
       requestBody:
         content:
           application/json:
             schema:
               type: array
               items:
-                $ref: '#/components/schemas/ProductPriceAggregate'
+                $ref: '#/components/schemas/FeedPricebook'
             examples:
-              ProductPriceWithDiscount:
-                summary: Product with discount example
-                description: Register product with $10 as the regular price and $8 as the final price. Final Price = min (20% from $10, $9)
-                value:
-                  [
-                    {
-                      "sku": "final-price-example",
-                      "type": "SIMPLE",
-                      "customerGroupCode": "0",
-                      "websiteCode": "us",
-                      "regular": 10,
-                      "discounts": [
-                        {
-                          "code": "special_price",
-                          "percentage": 20
-                        },
-                        {
-                          "code": "catalog_rule",
-                          "price": 9
-                        }
-                      ],
-                      "deleted": false,
-                      "updatedAt": "2023-07-20T19:32:16+00:00"
-                    }
-                  ]
-              CompositeProductPrice:
-                summary: Composite product price example
-                description: Register a configurable product "t-shirt" with 3 variants "t-shirt-s" ($10) and "t-shirt-m" ($11), "t-shirt-l" ($12). Configurable product "t-shirt" will have min price as $10 and max price as $12
+              FeedWithPricebookInformation:
+                summary: Example of product price books feed
+                description: Enables the feed with product price books information
```

Suggest to change to:
```suggestion
                description: Create Price Book "dealer-north" with "USD" currency
```

+ And change id to "dealer-not..."

---

Done